test/linalg/bidiag.jl

```diff
@@ -27,7 +27,6 @@ srand(1)
         @test Bidiagonal(dv,ev,:U) != Bidiagonal(dv,ev,:L)
         @test_throws ArgumentError Bidiagonal(dv,ev,:R)
         @test_throws DimensionMismatch Bidiagonal(dv,ones(elty,n),:U)
-        @test_throws ArgumentError Bidiagonal(dv,ev)
```

Comment:
well it used to exist but only for the purpose of throwing an error, so if it accidentally were brought back (say an incorrect use of a default third argument?) that could be surprising